screens/ClerkLoginScreen.js

```diff
@@ -99,48 +70,59 @@ export default class ClerkLogin extends React.Component {
             }
           }
         );
+    }).catch(err => {
+      console.error("Error looking up clerk", err);
     });
   }
 
   // From SignUpScreen. Sign in function. It sets the user token in local storage
   // to be the fname + lname and then navigates to homescreen.
   _asyncSignin = async recordId => {
```

Comment:
We should probably rename this, and update the comments

Reply:
Done!